swarm/storage/chunker.go

```diff
@@ -483,8 +483,12 @@ func (s *LazyChunkReader) Seek(offset int64, whence int) (int64, error) {
 	case 1:
 		offset += s.off
 	case 2:
```

Comment:
Unrelated, but please use `io.SeekStart`, `io.SeekEnd` instead of hard-coded values.